### DISCUSSION TITLE
erroring in importing openzeppelin

### DISCUSSION THREAD

**Discussion by heheboii11** [Cyfrin/foundry-full-course-cu#2804] [Q&A] [answered]
Hey everyone, I am currently facing error in importing the ```openzeppelin```, I have installed ```openzepplein```, after that I am unable to see ```Security```
inside the ```contracts``` of``` openzeppelin```
<img width="804" alt="Screenshot 2024-11-04 at 3 29 23 PM" src="https://github.com/user-attachments/assets/97eec31c-1468-4cf8-baa6-50d68dc647e4">

**Comment by EngrPips** [comment, marked as answer]
Hello @heheboii11, check the `utils` folder. You will see the `ReentrancyGuard.sol` file there. They move stuff in the updated version of the repo I assume

**Reply by heheboii11** [reply]
Thank you @EngrPips, I don't know how to check if anything gets updated

**Reply by EngrPips** [reply]
You can just check their GitHub page and see when was their latest release That way, you can know when the code was updated and you can be confident that some things have changed in the code starting the last update.
